flang/test/Integration/OpenMP/parallel-private-reduction-worstcase.f90

```diff
@@ -0,0 +1,261 @@
+! RUN: %flang_fc1 -fopenmp -emit-llvm %s -o - | FileCheck %s
+
+! stress test cfg and builder insertion points in mlir-to-llvm conversion:
```

Comment:
I don't think this test matches what's described under https://en.wikipedia.org/wiki/Stress_testing_(computing). For a compiler, I would understand sources that use more than a usual number of function/variable/identifiers, etc to test maybe scaling behavior. Four arrays is not yet an unusual amount. 

I think it could be called combinatorial testing.